content/release-10-18+/streaming-analytics-10-18+-bundle/10.18+_0.md

```diff
@@ -53,6 +53,11 @@ You can now create a selection list for an Analytics Builder template parameter.
 You can define selection lists for types such as string, float, source or destination, or geofence and you can also select a specific value to be the default value. The values that you define for a selection list are then available for selection when you create instances of the model.
 See also [Adding a selection list for a template parameter](https://cumulocity.com/docs/streaming-analytics/analytics-builder/#adding-a-selection-list-for-a-template-parameter) in the user documentation.
 
+#### Missing subassets in a group
+
+Previously, an error was thrown for an active model if a subasset of a group no longer existed in the inventory, for example, because a device was deleted.
+As of this version, missing subassets in a group are ignored and an error is no longer thrown.
```

Comment:
Should we explicityly mention that of the group or asset is empty as a result of device/asset deletion the error would still be raised?